templates/github/.github/workflows/scripts/pypi_tools.py.j2

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
Where is this file used?

Reply:
I was going to re-use this for the client package also, but then I decided not to do it so that I could keep the code similar for python client and ruby client. I could move this code back into create_tag_and_build_package.py.j2  if you think that will be better.

Reply:
I am guessing create_tag_and_build_package.py.j2 went away?